spec/controllers/events_controller_spec.rb

```diff
@@ -1,6 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe EventsController, type: :controller do
+  let(:admin_person) { FactoryGirl.build(:person, :admin) }
+
+  before do
+    allow_any_instance_of(EventsController).to receive(:pundit_user)
+      .and_return(admin_person)
+  end
```

Comment:
I recall you mentioning that you added the before block in to ensure that the user you created in factory girl had the correct authorizations for the spec. Is there a way in rspec to describe your before block such that it reads a little more semantically?